pyServer/manifests/linux/hdinsight

```diff
@@ -1,45 +0,0 @@
-echo,### HDInsight Logs ###
```

Comment:
I think you deleted this file either by accident or you didn't pull again from 'master' since this was just merged this morning. Can you fix this?